ush/machine/noaacloud.yaml

```diff
@@ -14,17 +14,18 @@ platform:
   RUN_CMD_UTILS: mpiexec -np $nprocs
   SCHED_NATIVE_CMD: --export=NONE
   PRE_TASK_CMDS: '{ ulimit -s unlimited; ulimit -a; }'
-  TEST_EXTRN_MDL_SOURCE_BASEDIR: /contrib/EPIC/UFS_SRW_App/develop/input_model_data
-  TEST_PREGEN_BASEDIR: /contrib/EPIC/UFS_SRW_App/develop/FV3LAM_pregen
-  TEST_ALT_EXTRN_MDL_SYSBASEDIR_ICS: /contrib/EPIC/UFS_SRW_App/develop/dummy_FV3GFS_sys_dir
-  TEST_ALT_EXTRN_MDL_SYSBASEDIR_LBCS: /contrib/EPIC/UFS_SRW_App/develop/dummy_FV3GFS_sys_dir
-  FIXaer: /contrib/EPIC/UFS_SRW_App/develop/fix/fix_aer
-  FIXgsm: /contrib/EPIC/UFS_SRW_App/develop/fix/fix_am
-  FIXlut: /contrib/EPIC/UFS_SRW_App/develop/fix/fix_lut
-  FIXorg: /contrib/EPIC/UFS_SRW_App/develop/fix/fix_orog
-  FIXsfc: /contrib/EPIC/UFS_SRW_App/develop/fix/fix_sfc_climo
-  FIXshp: /contrib/EPIC/UFS_SRW_App/develop/NaturalEarth
-  FIXgsi: /contrib/EPIC/UFS_SRW_App/develop/fix/fix_gsi
+  TEST_EXTRN_MDL_SOURCE_BASEDIR: /contrib/EPIC/UFS_SRW_data/develop/input_model_data
+  TEST_PREGEN_BASEDIR: /contrib/EPIC/UFS_SRW_data/develop/FV3LAM_pregen
+  TEST_ALT_EXTRN_MDL_SYSBASEDIR_ICS: /contrib/EPIC/UFS_SRW_data/develop/dummy_FV3GFS_sys_dir
+  TEST_ALT_EXTRN_MDL_SYSBASEDIR_LBCS: /contrib/EPIC/UFS_SRW_data/develop/dummy_FV3GFS_sys_dir
+  FIXaer: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_aer
+  FIXgsi: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_gsi
+  FIXgsm: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_am
+  FIXlut: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_lut
+  FIXorg: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_orog
+  FIXsfc: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_sfc_climo
+  FIXshp: /contrib/EPIC/UFS_SRW_data/develop/NaturalEarth
+  FIXgsi: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_gsi
   EXTRN_MDL_DATA_STORES: aws nomads
 data:
   ics_lbcs:
```

Comment:
Suggest to modify the subsequent line:


    FV3GFS:
       nemsio: /contrib/EPIC/UFS_SRW_data/develop/input_model_data/FV3GFS/nemsio/${yyyymmdd}${hh}
       grib2: /contrib/EPIC/UFS_SRW_data/develop/input_model_data/FV3GFS/grib2/${yyyymmdd}${hh}

Reply:
done.